src/Utilities/Compiler/Extensions/ISymbolExtensions.cs

```diff
@@ -193,59 +193,6 @@ public static ImmutableArray<IParameterSymbol> GetParameters(this ISymbol? symbo
                 allowedVisibilities.Contains(symbol.GetResultantVisibility());
         }
 
-        /// <summary>
-        /// Returns true if the given source symbol has been configured to be excluded from analysis by options.
-        /// </summary>
-        public static bool IsConfiguredToSkipAnalysis(
```

Comment:
Moved to `AnalyzerOptionsExtensions.cs`.

Reply:
@Evangelink Can we also move `MatchesConfiguredVisibility` extension method above to the same place? This way all options related methods are consistently extension methods on AnalyzerOptions instead of ISymbol.

Reply:
Note `MatchesConfiguredVisibility` has 2 overloads.

Reply:
Done